### ISSUE TITLE
About training and testing 

### ISSUE THREAD

**Issue by xwg0508** [Francis-Rings/ILA#10]
During training, I replaced the validation set with the test set to obtain the best.pth and the corresponding top1-acc. However, when testing, the top1-acc obtained by reading the best.pth is different from the one obtained during training. What is the reason for this?
